btk/plot_utils.py

```diff
@@ -95,6 +97,8 @@ def plot_blends(blend_images, blend_list, detected_centers=None,
         band_indices (list, default=[1,2,3]): list of length 3 with indices of
```

Comment:
Change band_indices  default in doctoring as well

Reply:
sounds good, thanks!